package.json

```diff
@@ -157,14 +157,18 @@
     "release:bump-version": "node ./scripts/release/bump-versions.js",
     "release:publish": "node ./scripts/release/publish.js",
     "switch-to-dev-dev": "sh ./scripts/switch-to-dev-dev.sh",
-    "clean-up-svg-icons": "sh ./scripts/clean-up-svg-icons.sh"
+    "clean-up-svg-icons": "sh ./scripts/clean-up-svg-icons.sh",
+    "check-dependencies": "node node_modules/@ckeditor/ckeditor5-dev-tests/bin/check-dependencies.js"
```

Comment:
After executing the command without additional arguments, it ends without any message:

```
yarn run check-dependencies
yarn run v1.22.10
$ node node_modules/@ckeditor/ckeditor5-dev-tests/bin/check-dependencies.js
✨  Done in 0.81s.
```

But it can be improved in the future, as a follow-up. We have two ways to solve the issue:

1. Exit with an error if packages aren't specified (easy to fix).
2. Check all packages found in the `package/` directory.

I'd go with the latter one.

Reply:
=> https://github.com/ckeditor/ckeditor5/issues/8861

Reply:
I didn't notice that you have already created an issue for this improvement and I did that too: https://github.com/ckeditor/ckeditor5/issues/8862 (with a few other minor improvements). I also created a new PR to fix them.